2-new-project/index.md

```diff
@@ -111,8 +111,8 @@ Once you add your own text, save the file in Sublime (hint: Cmd + S is the keybo
 Hint: make sure Jekyll is running in the Terminal (remember <code>jekyll serve</code>?). If there's anything wrong with the code, the Terminal will also show you potential errors, so check your Terminal window whenever the website doesn't load.
 </div>
 
-# What did you learn?
-
+<div class="recap">
 Wow, you have your own website now! You created it by starting a new Jekyll project through the Terminal (`jekyll new my-website`) and then running the website with `jekyll serve`. You were then able to open the website in your web browser, and edit the title, email and description on the front page. 
```

Comment:
When you wrap Markdown text in divs, the Markdown syntax is ignored, so you have to manually add `<code>` tags in place of backticks `